examples/waffle/waffle-ovm.json

```diff
@@ -1,5 +1,5 @@
 {
-  "compilerVersion": "./node_modules/@eth-optimism/solc",
+  "compilerVersion": "./../../node_modules/@eth-optimism/solc",
```

Comment:
Are you sure about this? I think it depends on the exact version of solc and whether or not it gets hoisted

Reply:
Yes, I'm sure. It gets hoisted now because I removed the `@eth-optimism/solc` import from `@eth-optimism/contracts`.